### ISSUE TITLE
Logger Bug - always throws when % is included in message

### ISSUE THREAD

**Issue by pjerryhu** [statsig-io/python-sdk#13]
This logger that hijacked our app logger will always throw exception when our `logger.warning("xxx%xx")` message includes a `"%"`

https://github.com/statsig-io/python-sdk/blob/main/statsig/utils.py#L43

**Comment by tore-statsig** [comment]
Thanks we will take a look!

**Comment by kenny-statsig** [comment]
Are you importing this logger and using it directly? I don't see how this could be "hijacking" your internal logger unless at some point you are either assigning the "statsig.sdk" logger to your logger or calling the "statsig.sdk" logger directly.

But I did test this out and was able to repro the exception "xxx%xxx". So we will fix this on our end.

```
from statsig import statsig

class _OutputLogger(Logger):
    _logs = defaultdict(list)
    def __init__(self, name='test-logger', level=logging.NOTSET):
        super().__init__(name=name, level=level)
        self.root.setLevel(level)

    def warning(self, msg, *args, **kwargs):
        self._logs["warning"].append(msg % args)
        super().warning(msg, *args, **kwargs)

    def clear_log_history(self):
        self._logs = defaultdict(list)


logging.setLoggerClass(_OutputLogger)
logging.basicConfig()
logger = logging.getLogger('test-logger')
logger.warning("test%warning")
```
Output:
```
kennyyi@Kennys-MBP test-statsig % python3 test-python-sdk.py
WARNING:test-logger:test warning
```

**Comment by pjerryhu** [comment]
Probably some importing issues. We created a `statsig_wrapper.py` to wrap some statsig calls, and have logging override there: `logging.setLoggerClass(logging.Logger)`.  I think this may get mixed up with the underlying statsig logger override

@kenny-statsig when would you expect the fix to be live?

**Comment by kenny-statsig** [comment]
Could you please share a code snippet?
I am not able to reproduce this issue with this:
```
import logging
import statsig

logging.basicConfig()
logging.warning("test%warning")
```
nor with this:
```
import logging
import statsig

logging.basicConfig()
logger = logging.getLogger()
logger.warning("test%warning")
```

**Comment by pjerryhu** [comment]
Oh I thought you said you were able to repro the `%` exception. Anyway, we wrap the statsig sdk call with a wrapper `statsig_wrapper.py`

here is the code snippet.
The init_statsig gets called in our app context setup. And it has been throwing in different processors. 

```
import logging
import os
from sentry_sdk import capture_exception
from statsig import statsig, StatsigOptions, StatsigEnvironmentTier, StatsigUser

def init_statsig():
    sdk_key = os.environ.get("STATSIG_SERVER_SECRET_KEY")
    env = os.environ.get("ENVIRONMENT", "development").lower()
    if env == "production":
        tier = StatsigEnvironmentTier.production
    else:
        tier = StatsigEnvironmentTier.development
    options = StatsigOptions(tier=tier)
    statsig.initialize(sdk_key, options)

def get_dynamic_config(config_name):
    ......
    try:
        res = statsig.get_config(StatsigUser(consumer_name), config_name).get_value()
        return res
    except Exception as e:
        capture_exception(e)
        return {}
```

**Comment by tore-statsig** [comment]
Are you importing your logger from statsig, or from logging?  @kenny-statsig any other thoughts?

**Comment by pjerryhu** [comment]
I'm importing from logging.

**Comment by bvand** [comment]
I'm seeing this as well and able to reproduce with this:
```
import logging

from statsig import statsig

logger = logging.getLogger(__name__)

if __name__ == "__main__":
    logger.info("hello %")
```
output of `python test.py`:
```
Traceback (most recent call last):
  File "/Users/***/test.py", line 8, in <module>
    logger.info("hello %")
  File "/Users/***/lib/python3.9/site-packages/statsig/utils.py", line 39, in info
    self._logs["info"].append(msg % args)
ValueError: incomplete format
```
changing the import order doesn't help

**Comment by kenny-statsig** [comment]
Thank you for sharing the repro. I will take another look into this.

**Comment by kenny-statsig** [comment]
The issue has been fixed in the last version 0.16.2 https://pypi.org/project/statsig/0.16.2/

**Comment by bvand** [comment]
May need a separate issue for this, but the statsig logger still seems to be hijacking the logging:
```
from statsig import statsig

import logging


logger = logging.getLogger(__name__)
logging.basicConfig(level=logging.INFO)

if __name__ == "__main__":
    logger.debug("hello %")
```
outputs
```
DEBUG:__main__:hello %
```
which is not what I would expect

**Comment by kenny-statsig** [comment]
^ Fixed in 0.16.3 https://pypi.org/project/statsig/0.16.3/
